mkdocs/docs/index.md

```diff
@@ -1,22 +1,156 @@
 # IREE
 
-** * UNDER CONSTRUCTION ** *
+IREE (**I**ntermediate **R**epresentation **E**xecution **E**nvironment[^1]) is
+an [MLIR](https://mlir.llvm.org/)-based end-to-end compiler that lowers Machine
+Learning (ML) models to a unified IR optimized for real-time inference on
+mobile/edge devices against heterogeneous hardware accelerators. IREE also
+provides flexible deployment solutions for its compiled ML models.
 
-## Overview
+## Introduction
 
-<!-- TODO(??): copy introduction from README.md -->
-<!-- TODO(??): describe this website? -->
+### Key features
 
-## Components
+- [x] Support for dynamic shapes, flow control, streaming, and other advanced
+model features
+- [x] Ahead-of-time compilation of scheduling and execution logic together
```

Comment:
I would place the 2nd as the 1st bullet as it's arguably the most distinctive feature we have. It's about the architecture. Then we can talk about great support for models (which is the 1st bullet right now), and the 3rd point I think we can cover about great target support (something along the line of "Support for various platforms and various CPU/GPU architectures). (I know we are talking about that in the next section, but that's expanding with more details.) Then we can talk about performance (power/resource efficiency and binary size, etc.). To me that's from a top-down model-to-deployment perspective WDYT?

Reply:
Ordering SGTM. I tried a few variations on wording and settled on

> Optimized for many CPU and GPU architectures

Phrasings like

> Compiler and runtime optimized for many CPU and GPU architectures

> Cross platform runtime with optimizations for many CPU and GPU architectures

> Cross platform compiler and runtime optimized for many CPU and GPU architectures

don't quite work for me, since they conflate a few things. Our runtime is "optimized" in the sense that it does as little as possible - we aim to do as much as possible ahead of time in the compiler. The "runtime" itself also doesn't even need to be used at all :P. The compiler is "optimized" in the sense that the compiler itself runs quickly on the host, but also in the sense that it generates efficient code for targets.